### PR TITLE
docs: adds section about configuring rules

### PR DESCRIPTION
**Pull request by bredamc**

### CODE REVIEW

**Comment by bredamc** [comment]
@wtrocki @craicoverflow @rkpattnaik780 This PR is now ready for review.

I added a new attribute, so 17 files were updated accordingly. However, the main changes are in the following file:
docs/registry/rhoas-cli-getting-started-registry/README.adoc

**Comment by bredamc** [comment]
@rkpattnaik780 Can you please review this PR too? Many thanks :)

File: docs/registry/rhoas-cli-getting-started-registry/README.adoc [hunk 1]
```diff
@@ -373,6 +375,103 @@ For more information about any of the options, view the command help `rhoas serv
...
+
+[source,shell]
+----
+$ rhoas service-registry rule disable --rule-type=validity --artifact-id=my-artifact
+----
+--
+
+[NOTE]
+====
+You can use additional options, such as `--group` and `--instance-id`, to specify the rule to update.
```
Comment (rkpattnaik780) [inline docs/registry/rhoas-cli-getting-started-registry/README.adoc:470]: `--group` and `--instance-id` allow us to select the atifact/registry instance. I am thinking if we can update it to something like "specify entity in which rules are to be updated". Wdyt?

**Review by rkpattnaik780 — APPROVED** [review]
LGTM!

**Comment by bredamc** [comment]
Thank you @rkpattnaik780 !

@smccarthy-ie Ready for your peer review now, please and thank you :)

File: docs/registry/rhoas-cli-getting-started-registry/README.adoc (hunk 1 above)
Comment (bredamc) [reply docs/registry/rhoas-cli-getting-started-registry/README.adoc:470]: How about "to specify the entity that is governed by the updated rule"?

File: docs/registry/rhoas-cli-getting-started-registry/README.adoc [hunk 2]
```diff
@@ -373,6 +375,103 @@ For more information about any of the options, view the command help `rhoas serv
 ====
 --
 
+[id="proc-configuring-service-registry-rules_{context}"]
+== Configuring {registry} rules
+
+[role="_abstract"]
+To prevent invalid content from being added to {registry}, you can configure optional rules to check the artifact content. Global rules apply to all artifacts in a particular {registry} instance. Configured artifact-specific rules override any configured global rules. Before a new artifact version can be uploaded to the registry, all configured global rules or artifact-specific rules must pass.
+
```
Comment (smccarthy-ie) [inline docs/registry/rhoas-cli-getting-started-registry/README.adoc:383]: Suggest "Global rules apply to all artifacts in a particular {registry} instance.  Artifact rules apply to the specified artifact only. Configured artifact rules override any configured global rules..." to match the current UI and docs.

**Comment by smccarthy-ie** [comment]
Nice work @bredamc, looks great! Left a comment about defining "artifact rules" as artifact-specific up front.

File: docs/registry/rhoas-cli-getting-started-registry/README.adoc (hunk 1 above)
Comment (smccarthy-ie) [reply docs/registry/rhoas-cli-getting-started-registry/README.adoc:470]: Suggest "to specify the {registry} group or instance to which the rules apply."

Comment (bredamc) [reply docs/registry/rhoas-cli-getting-started-registry/README.adoc]: Done :)

File: docs/registry/rhoas-cli-getting-started-registry/README.adoc (hunk 2 above)
Comment (bredamc) [reply docs/registry/rhoas-cli-getting-started-registry/README.adoc:383]: Done :)

**Pull request merged**